### PR TITLE
[AP-1441] remove tap-adwords

### PR DESCRIPTION
**Pull request by amofakhar**
## Problem

There is a vulnerable dependency alert about setuptools < 65.5.1 .
Currently we are using version <=57.0.5 because of tap-adwords 1.11.2 .  googleads 17.0.0  is one of its requirements which works only with that old setuptools !
In the latest release of tap-adwords (1.13.1) they are using googleads 17.0.0 yet, since 37.0.0  is the latest version!
In my opinion we can remove tap-adwords from PPW for now. Because:

This VA is marked as high by github.

We don’t have any type of this tap running in PPW cluster.

tap-adwords repository is not very active and the lates release backs to Oct 2021

## Proposed changes

remove `tap-adwords`

## Types of changes

What types of changes does your code introduce to PipelineWise?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] Description above provides context of the change
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Unit tests for changes (not needed for documentation changes)
- [x] CI checks pass with my changes
- [ ] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [x] Commit message/PR title starts with `[AP-NNNN]` (if applicable. AP-NNNN = JIRA ID)
- [x] Branch name starts with `AP-NNN` (if applicable. AP-NNN = JIRA ID)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions

### CODE REVIEW

File: docs/connectors/taps.rst [hunk 1]
```diff
@@ -109,26 +109,6 @@ PipelineWise can replicate data from the following data sources:
...
-.. container:: tile-wrapper
-
-    .. container:: tile
-
-        .. container:: img-hover-zoom
-
-          .. image:: ../img/jira-logo.png
-             :target: taps/jira.html
-
-        :ref:`tap-jira`
```
Comment (teaf-wise) [inline docs/connectors/taps.rst:121]: tap-jira removal also necessary?

Comment (amofakhar) [reply docs/connectors/taps.rst]: 🙈It was by mistake! fixed now!

File: README.md [hunk 2]
```diff
@@ -59,8 +59,6 @@ consumes data from taps and do something with it, like load it into a file, API
...
 | Tap       | **[Jira](https://github.com/singer-io/tap-jira)** | | [![PyPI version](https://badge.fury.io/py/tap-jira.svg)](https://badge.fury.io/py/tap-jira) | Extracts data from Atlassian Jira using Base auth or OAuth credentials |
 | Tap       | **[MongoDB](https://github.com/transferwise/pipelinewise-tap-mongodb)** | | [![PyPI version](https://badge.fury.io/py/pipelinewise-tap-mongodb.svg)](https://badge.fury.io/py/pipelinewise-tap-mongodb) | Extracts data from MongoDB databases. Supporting Log-Based and Full Table replications |
-| Tap       | **[AdWords](https://github.com/singer-io/tap-adwords)** | Extra | [![PyPI version](https://badge.fury.io/py/tap-adwords.svg)](https://badge.fury.io/py/tap-adwords) | Extracts data Google Ads API (former Google Adwords) using OAuth and support incremental loading based on input state |
-| Tap       | **[Google Analytics](https://github.com/transferwise/pipelinewise-tap-google-analytics)** | Extra | [![PyPI version](https://badge.fury.io/py/pipelinewise-tap-google-analytics.svg)](https://badge.fury.io/py/tap-adwords) | Extracts data from Google Analytics |
```
Comment (teaf-wise) [inline README.md:63]: Is the tap from google analytics entry needed for deletion?

Comment (amofakhar) [reply README.md:63]: I searched all `tap-adwords` and this one was by mistake :)

**Review by teaf-wise — APPROVED** [review]

**Pull request merged**